examples/docs_snippets/docs_snippets_tests/guides_tests/managing_ml_tests/managing_ml_tests.py

```diff
@@ -0,0 +1,32 @@
+# isort: skip_file
+
+from dagster._core.definitions.load_assets_from_modules import assets_from_modules
+from dagster._core.definitions.materialize import materialize
+from docs_snippets.guides.dagster.managing_ml.managing_ml_code import (
```

Comment:
instead of listing these individually, you can do

```python
from docs_snippets.guides.dagster.managing_ml import managing_ml_code

...


assets = load_assets_from_modules([managing_ml_code])
materialize(assets)
```

Reply:
updated